Makefile

```diff
@@ -30,6 +30,7 @@ release: \
 
 check:
 	@gofmt -l -s $(GOFILES) | read; if [ $$? == 0 ]; then gofmt -s -d $(GOFILES); exit 1; fi
+	@which terraform &> /dev/null && terraform fmt -check ; if [ ! $$? -eq 0 ]; then exit 1; fi
```

Comment:
nit: I don't think this should be conditional. it should just fail if you don't have terraform in your path

Reply:
fixed... and rebased